src/Ombi.Schedule/Jobs/Plex/PlexAvailabilityChecker.cs

```diff
@@ -174,6 +174,8 @@ private async Task ProcessTv(List<ChildRequests> tv)
                     };
                     notification.Substitutes.Add("Season", availableEpisode.First().SeasonNumber.ToString());
                     notification.Substitutes.Add("Episodes", string.Join(", " ,availableEpisode.Select(x => x.EpisodeNumber)));
+                    notification.Substitutes.Add("EpisodesCount", $"{availableEpisode.Count}");
+                    notification.Substitutes.Add("SeasonEpisodes", string.Join(", ", availableEpisode.Select(x => $"{x.SeasonNumber}x{x.EpisodeNumber}" )));
```

Comment:
It would be worth encapsulating this logic to reuse in all of the different places

Reply:
I can't agree more, but the scope is larger than those two lines of code. I think all this block is worth encapsulating, if not more: https://github.com/sephrat/Ombi/blob/partially-available-notification/src/Ombi.Schedule/Jobs/Plex/PlexAvailabilityChecker.cs#L147-L180  
Would you like me to work on it in this PR or should we separate this into another PR?

Reply:
Yeah that's fair! 

Let's do that in a separate PR, in that case, to keep things small and easier to review!